app/views/custom/verification/residence/new.html.erb

```diff
@@ -0,0 +1,80 @@
+<div class="verification account row">
+  <% track_event(category: "verification", action: "start_census") %>
+  <div class="small-12 column">
+
+    <%= back_link_to account_path, t("verification.back") %>
+
+    <h1 class="d-block"><%= t("verification.residence.new.title") %></h1>
+
+    <div class="user-permissions small-12">
+
+      <p><%= t("verification.user_permission_info_budgets") %></p>
+      <p><%= raw t("verification.residence.new.help_needed",
+            email: mail_to(Rails.configuration.deploy['support_email'])
+          ) %>
+      </p>
+      <hr/>
+
+    </div>
+
+    <%= form_for @residence, as: "residence", url: residence_path do |f| %>
+
+      <%= render "errors" %>
+
+      <div class="row">
+        <div class="small-12 medium-4 column">
+
+          <div class="inline-block">
+            <%= f.label t("verification.residence.new.document_number") %>
+          </div>
+
+          <button type="button" class="inline-block" data-toggle="info-document-number">
+            <span class="icon-help"></span>
+            <span class="show-for-sr"><%= t("verification.residence.new.document_number_help_title") %></span>
+          </button>
+
+          <div class="dropdown-pane" id="info-document-number" data-dropdown
+                data-hover="true" data-hover-pane="true">
+            <%= sanitize(t("verification.residence.new.document_number_help_text")) %>
+          </div>
+
+          <%= f.text_field :document_number, label: false %>
+        </div>
+      </div>
+
+      <div class="row">
+        <%= fields_for @sms do |s| %>
+
+          <div class="small-12 medium-4 column">
+            <%= s.label :phone, t("verification.sms.new.phone"), class: "inline-block" %>
+            <p class="help-text" id="phone-text-help"><%= t("verification.sms.new.phone_note") %></p>
+            <%= s.telephone_field :phone, label: false,
+                                  aria: { describedby: "phone-help-text" }
+                                  %>
+          </div>
+        <% end %>
+      </div>
+
+      <div class="small-12 mt-3">
+        <%= f.check_box :terms_of_service,
+          label: t("verification.residence.new.accept_terms_adult") %>
+
+        <%= f.check_box :adult,
+          label: t("verification.residence.new.accept_terms_resident",
+                  residence: Rails.configuration.deploy['city']) %>
+
+        <%= f.check_box :resident,
+          label: t("verification.residence.new.accept_terms_terms",
+                  terms_url: link_to(t("verification.residence.new.terms"), "/census_terms",
```

Comment:
@tsundokul what is `/census_terms`?

Reply:
Looking at the footer, I think `/conditions` is the actual terms and conditions page?

```suggestion
                  terms_url: link_to(t("verification.residence.new.terms"), "/conditions",
```

cc @catileptic it's mostly empty BTW:

![image](https://user-images.githubusercontent.com/1650875/132362772-6994d04f-743b-4e1f-a2c5-a2d60300b018.png)